src/mpi/pt2pt/mpir_request.c

```diff
@@ -569,8 +570,16 @@ int MPIR_Grequest_progress_poke(int count,
             {
 		mpi_errno = (request_ptrs[i]->u.ureq.greq_fns->poll_fn)(request_ptrs[i]->u.ureq.greq_fns->grequest_extra_state, &(array_of_statuses[i]));
                 if (mpi_errno) MPIR_ERR_POP(mpi_errno);
+            if MPIR_Request_is_complete(request_ptrs[i])
+                made_progress = 1;
```

Comment:
Formatting should match the surrounding code.

Reply:
fixed I believe after switching from space to tab